### DISCUSSION TITLE
Setting up local use for a raspberry pi

### DISCUSSION THREAD

**Discussion by Eirikr70** [SEPIA-Framework/sepia-docs#142] [Q&A]
Hello, I have set up a sepia server on my raspberry pi. I can access it through [http://sepia-home.local:20721/app/index.html](url). It seems to work fine (although it doesn't speak french ;)). I have a mic and a speaker on the pi itself, in order to make it a smart speaker. Is there a way for me to use them instead of the url ?

**Comment by fquirin** [comment]
Hi @Eirikr70 ,
have you seen the DIY client? :slightly_smiling_face:  https://github.com/SEPIA-Framework/sepia-installation-and-setup/tree/master/sepia-client-installation/rpi
Depending on your Raspberry Pi you can run server and client on the same machine, but I recommend a Pi4 4GB for that. 

> although it doesn't speak french ;)

I'd love to support more languages but the NLU is tricky. SEPIA's intent recognition and smart-services are heavily optimized by hand for English and German. The latest SEPIA version has opened up a bit more to support new languages and I'll try to further improve it step by step.

**Comment by Eirikr70** [comment]
Hadn't found it. Going to try that way. Danke sehr !
About french, I'm conscious that adding a new language is not just a question of adding a code in a table. You're doing a great job !
